internal/pkg/service/common/etcdop/op/retry.go

```diff
@@ -27,6 +27,11 @@ var retriedErrCodes = map[codes.Code]bool{
 	codes.FailedPrecondition: true,
 }
 
+// nolint: gochecknoglobals
+var retriedErrs = map[error]bool{
+	rpctypes.ErrFutureRev: true, // broken etcd instance may be frozen in some old revision
```

Comment:
Nechceme robit retries na vsetky `OutOfRange` error code chyby, ale iba na tuto (`mvcc: required revision is a future revision`).